cmd/controller/controller-runtime-main.go

```diff
@@ -72,9 +81,10 @@ func controllerRuntimeStart() error {
 		eventtype.ProvideController,
 		feed.ProvideController,
 		flow.ProvideController,
-		subscription.ProvideController,
 	}
 
+	providers = addExperimentalControllers(logger, experimental, providers)
```

Comment:
Ignorable nit: I think this might read better as

```go
providers = append(providers, getExperimentalControllerProviders(logger, experimental))
```

So the reader knows right away that the experimental controllers are being appended to the list.